<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GyroDialog</class>
 <widget class="QDialog" name="GyroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gyro Serial Config </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    color:rgb(196, 160, 0);
    background-color: rgb(40, 40, 40);
}

QWidget {
    color:rgb(196, 160, 0);
    background-color: rgb(40, 40, 40);
}

QWidget::disabled {
    color:rgb(0, 0, 0);
    background-color: rgb(40, 40, 40);
}

QGroupBox {
    border: 1px solid gray;
    border-color: rgb(196, 160, 0);
    margin-top: 29px;
    font-size: 16px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QLineEdit::disabled {
    color:rgb(0, 0, 0);
}

QToolTip {
    color:rgb(196, 160, 0);
    background-color: rgb(40, 40, 40);
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    color: rgb(255, 255, 0);
        background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
}

QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: yellow;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QComboBox QAbstractItemView {
    border: 1px solid black;
    selection-background-color: yellow;
    selection-color: black;
}
QLineEdit
{
    border: 1px solid gray;
}

QCheckBox {
	background-color: rgb(85, 87, 83);
  border: 2px solid #5A5A5A;
  spacing: 4px;
  outline: none;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 4px;
}

QCheckBox::indicator {
  margin-left: 0px;
  height: 16px;
  width: 16px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Serial Config</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Port Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="portNameCombo">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22">
              <item>
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>Baud Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="baudRateCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_23">
              <item>
               <widget class="QLabel" name="label_26">
                <property name="text">
                 <string>Stop Bits</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="stopBitsCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_24">
              <item>
               <widget class="QLabel" name="label_37">
                <property name="text">
                 <string>Data Bits</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="dataBitsCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_25">
              <item>
               <widget class="QLabel" name="label_38">
                <property name="text">
                 <string>Parity</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="parityCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSaveSerialGyro">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>TCP Config</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_27">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_28">
              <item>
               <widget class="QLabel" name="label_39">
                <property name="text">
                 <string>Tcp/IP</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_29">
              <item>
               <widget class="QLabel" name="label_40">
                <property name="text">
                 <string>Port</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>62</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
